options.go

```diff
@@ -15,87 +15,69 @@
 package greptime
 
 import (
+	"crypto/tls"
 	"time"
 
 	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
 	"google.golang.org/grpc/credentials/insecure"
 	"google.golang.org/grpc/keepalive"
 )
 
 var (
-	uaOpt = grpc.WithUserAgent("greptimedb-ingester-go/" + Version)
-
-	// TODO(yuanbohan): SecurityOptions
+	uaOpt       = grpc.WithUserAgent("greptimedb-ingester-go/" + version)
+	secureOpt   = grpc.WithTransportCredentials(credentials.NewTLS(&tls.Config{InsecureSkipVerify: false}))
 	insecureOpt = grpc.WithTransportCredentials(insecure.NewCredentials())
-
-	defaultKeepaliveInterval = 30 * time.Second
-	defaultKeepaliveTimeout  = 5 * time.Second
 )
 
-type Options struct {
-	keepalive *KeepaliveOption
+type options struct {
+	secure    *grpc.DialOption
+	keepalive *grpc.DialOption
 }
 
-func NewOptions(keepalive *KeepaliveOption) *Options {
-	return &Options{
-		keepalive: keepalive,
+func newOptions() *options {
+	return &options{
+		secure: &insecureOpt,
 	}
 }
 
-func (o *Options) WithKeepalive(keepalive *KeepaliveOption) *Options {
-	o.keepalive = keepalive
-	return o
-}
-
-func (o *Options) Build() []grpc.DialOption {
-	options := []grpc.DialOption{uaOpt, insecureOpt}
-
-	if o == nil {
-		return options
+func (o *options) withKeepalive(interval, timeout time.Duration) *options {
+	if interval == 0 && timeout == 0 {
```

Comment:
The default value of `time.Duration` type is 0s?  It can be show by the following code: 
```
var duration time.Duration
fmt.Printf("Default duration value: %v\n", duration) //Default duration value: 0s
```

Reply:
```
// A Duration represents the elapsed time between two instants
// as an int64 nanosecond count. The representation limits the
// largest representable duration to approximately 290 years.
type Duration int64
```

the code above is from time package. The zero value of Duration is 0? or is there a better way to tell the zero value of Duration?

Reply:
This default is 0, ignore it.